pyserini/prebuilt_index_info.py

```diff
@@ -1887,6 +1887,398 @@
         "downloaded": False
     },
 
+    # BEIR (v1.0.0) contriever indexes
+    "beir-v1.0.0-trec-covid-contriever": {
+        "description": "Faiss index for BEIR v1.0.0 (TREC-COVID) corpus encoded by Contriever encoder.",
+        "filename": "faiss.beir-v1.0.0-trec-covid.contriever.20230124.tar.gz",
+        "readme": "faiss.contriever.20230124.README.md",
```

Comment:
```suggestion
        "readme": "faiss.beir-v1.0.0.contriever.20230124.README.md",
```

We should add `beir-v1.0.0` because `contriever` might be applied to other datasets?

Reply:
changed the readme file name